player/lua/osc.lua

```diff
@@ -2195,11 +2195,13 @@ function update_margins()
 
     utils.shared_script_property_set("osc-margins",
         string.format("%f,%f,%f,%f", margins.l, margins.r, margins.t, margins.b))
+    utils.user_data_set("osc/margins", margins)
```

Comment:
I'd prefer the fields to be called `top`, `bottom`, `left`, `right` instead of just single letters.

Reply:
Do you feel particularly strongly about this? These are used all over osc.lua with the current names.

Reply:
We can also leave it as is, it's not that big a deal.
Anyone that ends up using that will probably look at the builtin scripts anyway and know what those letters stand for.